app.json

```diff
@@ -0,0 +1,10 @@
+{
+  "addons": [
+    "logentries:entry_plus"
```

Comment:
I would just specify "logentries" and let Heroku choose the free tier initially. That has the effect of allowing you to spin up a new environment with suitable dependencies - you can then choose to scale those as you see fit.